src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -92,7 +92,7 @@ class ArrayField extends React.Component {
           },
         );
         focusElement(`[name="${scrollElementName}"] ${focusElementSelector}`);
-      }, 100);
+      }, window.Mocha ? 0 : 100);
```

Comment:
it might be even safer to totally skip setting this timeout in the first place, but setting the timeout delay to 0 seems to work,

Reply:
I'm not 💯 keen on referencing our testing environment in our application code, though I do agree that it is a cleaner way of detecting if we are in a testing environment. I really appreciate you digging in here, how did you figure out this was causing the issue?

Reply:
Isn't line 87 essentially doing the same thing already, referencing code that's set in `mocha-setup.js`. And there is lots of other places where a unit test will set `window.VetsGov.pollTimeout` and that value will be checked in the application code.

I first replicated the failure locally by inserting the `await wait()` call in the test that broke in CI. Once I got the failure locally, I switched the mocha reporter to `unit` so I could easily see what tests had run right before the failure. The ArrayField test jumped out as being the culprit, based on the error output. I then confirmed that by running just those two test files. From there it was pretty easy to find the offending timeout.

Reply:
Did you try to set a const using something like `process.env`? That seems like a more recommended way than `window.mocha`.

https://github.com/department-of-veterans-affairs/vets-website/blob/6e8973b195f66d3a97ea1bc111185f1cab8b92d5/src/platform/forms-system/src/js/review/submit-states/GenericError.jsx#L33

And don't quote me on this, but webpack may be smart enough to identify dead branches using conditionals based on `process.env` and eliminate them from compiled code.

Reply:
This could be something to try. Webpack does have a couple of [plugins](https://webpack.js.org/plugins/environment-plugin/) that allow for eliminating conditionals based on process.env from the compiled code (if they don't match the environment specified), but I don't see those plugins in our webpack setup. This doesn't mean that our webpack setup doesn't do this, I only took a quick look, but we'd need to be careful with this approach. Also, our local and dev environments use LOCALHOST (I think, I know for sure that our local does, not sure about dev or review instances), so any conditional based on that environment might be applied in all of those places, not just the tests.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/91999f78239db0b2da1c4ae815e827c86c02c792/script/run-unit-test.js#L46-L48

The default is to use `mochaPath` rather than `coveragePath`, but we could probably add a `NODE_END=test` to `mochaPath` and then set the timeout const based on `process.env.NODE_ENV`. Or we could use the already provided `BABEL_ENV`, but that seems like it's less node-y